x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/description_step/helpers.tsx

```diff
@@ -218,21 +229,37 @@ export const buildUrlsDescription = (label: string, values: string[]): ListItems
       {
         title: label,
         description: (
-          <EuiFlexGroup gutterSize="none" alignItems="flexStart" direction="column">
-            {values.map((val: string) => (
-              <EuiFlexItem>
-                <ReferenceLinkItem
-                  href={val}
-                  target="_blank"
-                  iconType="link"
-                  size="xs"
-                  flush="left"
-                >
-                  {val}
-                </ReferenceLinkItem>
-              </EuiFlexItem>
-            ))}
-          </EuiFlexGroup>
+          <EuiText size="s">
+            <ul>
+              {values
+                .filter(v => !isEmpty(v))
+                .map((val: string, index: number) => (
```

Comment:
nit: You don't need `:string` and `:number` types here as that just duplicates existing types.

No changes requested as there is so much good work here we don't want to delay this any more.

Reply:
Ah! You'd mentioned that on a different one. I missed checking the rest of the code for it. I have one more PR for this coming up so I'll take note (no pun intended) and update it then.

Reply:
Ended up adding this to the PR as had to update circular dep check file.